packages/ckeditor5-dev-changelog/src/utils/removeemptydirs.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import fsExtra from 'fs-extra';
+import fs from 'fs/promises';
+import upath from 'upath';
+
+/**
+ * Removes empty directories from the filesystem.
+ * This function cleans up any empty directories that remain after processing.
+ */
+export async function removeEmptyDirs( directory: string ): Promise<void> {
+	if ( !fsExtra.existsSync( directory ) ) {
```

Comment:
Why not async version?

Please update also other files were sync version is used.

Reply:
There is no async await version for `existsSync` unfortunately. So I'm leaving it as it is for now.

Reply:
Changed to `pathExists`.